elasticsearch/connection/http_requests.py

```diff
@@ -118,6 +119,8 @@ def perform_request(
 
         start = time.time()
         request = requests.Request(method=method, headers=headers, url=url, data=body)
+        if not self.ssl_show_warn:
+            requests.packages.urllib3.disable_warnings()
```

Comment:
Hi, thank you for the patch, could you please move this to `__init__`, or is there a reason why it is here? This way we run it with every request which is not ideal. Thank you!

Reply:
Make sense, moved to __init__ and commits amended.